requirements_test_all.txt

```
@@ -1205,6 +1205,9 @@ pysqueezebox==0.5.5
 # homeassistant.components.syncthru
 pysyncthru==0.7.10
 
+# homeassistant.components.tankerkoenig
+pytankerkoenig==0.0.6
```

If you update the version above, it should also be updated here ;-)
```suggestion
pytankerkoenig==0.0.7
```

---

this will automatically be done by the hassfest script - in case I would bump the version now 🙂

---

see https://github.com/home-assistant/core/pull/68386#discussion_r831517185